tests/unit/test-cmd/test.lint.js

```diff
@@ -127,20 +126,22 @@ describe('lint', () => {
     });
   });
 
-  it('passes a file filter to the linter', () => {
-    const fileFilter = fake(new FileFilter());
-    const {lint, createLinter} = setUp({fileFilter});
-    return lint()
-      .then(() => {
-        assert.equal(createLinter.called, true);
-        const config = createLinter.firstCall.args[0].config;
-        assert.isFunction(config.shouldScanFile);
-
-        // Simulate how the linter will use this callback.
-        config.shouldScanFile('manifest.json');
-        assert.equal(fileFilter.wantFile.called, true);
-        assert.equal(fileFilter.wantFile.firstCall.args[0], 'manifest.json');
-      });
+  it('ensure linter build fileFilter and shouldScanFile correctly', () => {
```

Comment:
This test spec is hard for me to read. How about changing it to: "configures and passes a fileFilter to the linter"